### PR TITLE
📰 Blog post announcing board member changes

### PR DESCRIPTION
**Pull request by tim-schilling**
This can't be merged until we clean up the `<Need something here>` for Jen's thank you.

### CODE REVIEW

**Comment by VeldaKiara** [comment]
Hey @tim-schilling looks good to me.

File: _posts/2024-05-01-board-updates.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+show_related_posts: false
+title: "Board Member Update for May 2024"
+permalink: /announcements/2023/05/01/board-updates/
+---
+
+We have a new board member as of May 2024!
+
+## Outgoing Director
+
+We would like to take a moment to thank our outgoing director, Jennifer Myers for her contributions to our board during her years of service. Jen joined the board in 2021 and has been an invaluable asset to the board during her tenure as treasurer. <Need something here>. We are deeply grateful for the time and effort she has invested.
```
Comment (jefftriplett) [inline _posts/2024-05-01-board-updates.md]:
```suggestion
We want to take a moment to thank our outgoing director and Treasurer, Jennifer Myers, for her contributions to our board during her years of service. Jen joined the board in 2021 and has been an invaluable asset to the board during her tenure. <Need something here>. We are deeply grateful for the time and effort she has invested.
```

"<Need something here>" - did you have thoughts on direction?

Comment (tim-schilling) [reply _posts/2024-05-01-board-updates.md]: I saw Josue's thank-you, "Josue joined DEFNA in 2018, was a DjangoCon US volunteer, and was no stranger to speaking at DjangoCon US. Josue has always been a voice for pushing DEFNA forward and has hosted [our YouTube interviews](https://www.youtube.com/watch?v=KMO4RR5ISRE&list=PL2NFhrDSOxgUwF244g835zO3ciMkKILyJ) for many years." and felt like we should match that tone here.

Comment (jefftriplett) [reply _posts/2024-05-01-board-updates.md]: This isn't flowing, but maybe it helps with filling in some gaps: 

While Jen never attended DjangoCon US, she helped us balance our books, handle finances, processing grants during our first online and our return to in-person events.

File: _posts/2024-05-01-board-updates.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+show_related_posts: false
+title: "Board Member Update for May 2024"
+permalink: /announcements/2023/05/01/board-updates/
+---
+
+We have a new board member as of May 2024!
+
+## Outgoing Director
+
+We want to take a moment to thank our outgoing director and Treasurer, Jennifer Myers, for her contributions to our board during her years of service. Jen joined the board in 2021 and has been an invaluable asset to the board during her tenure. <Need something here>. We are deeply grateful for the time and effort she has invested.
```
Comment (PeterGrand) [inline _posts/2024-05-01-board-updates.md]:
```suggestion
We want to take a moment to thank our outgoing director and Treasurer, Jennifer Myers, for her contributions to our board during her years of service. Jen joined the board in 2021 and has been an invaluable asset to the board during her tenure. She spent many hours keeping the organization's finances in order and making sure all of our grant recipients and speakers received their reimbursements correctly. We are deeply grateful for the time and effort she has invested, and her presence on the board will be missed.
```

File: _posts/2024-05-01-board-updates.md (hunk 1 above)
Comment (tim-schilling) [reply _posts/2024-05-01-board-updates.md]: I think we should go with Peter's.

**Review by sodevious — APPROVED** [review]

**Pull request merged**